DepotDownloader/ContentDownloader.cs

```diff
@@ -1295,5 +1281,49 @@ private static async Task DownloadSteam3Async(uint appId, List<DepotDownloadInfo
             }
 
         }
+
+        static void DumpManifestToTextFile( DepotDownloadInfo depot, ProtoManifest manifest )
+        {
+            StringBuilder manifestBuilder = new StringBuilder();
+            string txtManifest = Path.Combine( depot.installDir, string.Format( "manifest_{0}_{1}.txt", depot.id, depot.manifestId ) );
+            manifestBuilder.Append( string.Format( "Content Manifest for Depot {0}\n\n", depot.id ) );

```

Comment:
Shouldn't these be platform newlines (`Environment.NewLine`)?

Reply:
Pretty sure newlines are written as platform specific when open file in text mode.

Reply:
If you have a StreamWriter and then use WriteLine methods, it will write platform-specific newlines.

If you shove `\n` into a string and then write the whole string at once, the runtime does not magically insert `\r` characters where it think it would be appropriate.

Reply:
Even Windows' notepad understands \n now, it's fine. Otherwise you can use `AppendLine`

Reply:
Notepad only understands it on the latest few feature release of Windows 10 and Windows Server 2019. Unfortunately Windows Server 2016 at least still doesn't understand `\n` alone :(

Reply:
...huh, I guess it's different in C#? In C++ and Python, writing ``\n`` to a file opened in text mode produces platform-specific newline.

Reply:
```csharp
> using System.IO;
> File.WriteAllText("rn.txt", "foo\nbar")
> File.ReadAllBytes("rn.txt")
byte[7] { 102, 111, 111, 10, 98, 97, 114 }
```

```python
>>> file = open('rn2.txt', 'w')
>>> file.write("foo\nbar")
7
>>> file.close()
>>> file = open('rn2.txt', 'rb')
>>> file.read()
b'foo\r\nbar'
>>> file.close()
>>> file = open('rn2.txt', 'r')
>>> file.read()
'foo\nbar'
>>> file.close()
```

From what I can tell, some Windows APIs will automatically normalize newlines (`open(..., O_TEXT)`) and some will not (`fopen`)?

It seems like a weird world, so better to be explicit about it.